linera-views/src/lib.rs

```diff
@@ -128,25 +128,40 @@ pub mod test_utils;
 /// Re-exports used by the derive macros of this library.
 #[doc(hidden)]
 pub use {
-    async_lock, async_trait::async_trait, futures, generic_array, linera_base::crypto, serde, sha3,
+    async_lock,
+    async_trait::async_trait,
+    futures, generic_array,
+    linera_base::crypto,
+    once_cell::sync::Lazy,
+    prometheus::{register_int_counter_vec, IntCounterVec},
+    serde, sha3,
 };
 
-/// Does nothing. Use the metrics feature to enable.
-#[cfg(not(feature = "metrics"))]
-pub fn increment_counter(_name: &str, _struct_name: &str, _base_key: &[u8]) {}
-
 /// Increments the metrics counter with the given name, with the struct and base key as labels.
-#[cfg(feature = "metrics")]
```

Comment:
@andresilva91 Remove the `cfg` here might have caused the performance regression.

Reply:
You think so, why? Is this supposed to do nothing in some cases?

Reply:
I'm confused. What's unclear? There was a feature to control metric production and now it's always on.

Reply:
I just don't understand why turning on metric production would break a test 😅 I'm probably lacking either Rust knowledge or knowledge of the system to get it. But regardless it seems like you're right, I'm verifying it now then will update the PR